doc/api_methods/AddList.md

```diff
@@ -17,3 +17,18 @@ An associative array which contains list data.
 | --- | --- | --- | --- |
 | name (required) | string | 90 chars | A name of the list. |
 | description (optional) | string\|null| 250 chars | A description of the list. |
+
+## Error handling
```

Comment:
This text is repeated in all docs, maybe we should create a separate document about error handling and move it there? Not sure whether to leave error codes where they are or to move them as well. If the codes of different methods don't collide then they shouldn't be confusing if organized on a single page, should they?

Reply:
Hmm, I was thinking about that and I decided to copy and paste it everywhere. The reason being is the convenience for the reader. So when they are working on a feature they can see everything in a single page without needing to go back and forth between different pages. 

> the codes of different methods don't collide

They sometimes do when they throw a similar exception, for example UnsubscribeFromLists and SubscribeToLists share codes.

So if we wanted to create a single page with all the codes we would have to separate the code by the function anyway, so I thought it is better to leave it here.